pkg/controller/admissionchecks/provisioning/constants.go

```diff
@@ -19,8 +19,8 @@ package provisioning
 const (
 	ConfigKind             = "ProvisioningRequestConfig"
 	ControllerName         = "kueue.x-k8s.io/provisioning-request"
-	ConsumesAnnotationKey  = "cluster-autoscaler.kubernetes.io/consume-provisioning-request"
-	ClassNameAnnotationKey = "cluster-autoscaler.kubernetes.io/provisioning-class-name"
+	ConsumesAnnotationKey  = "autoscaling.x-k8s.io/consume-provisioning-request"
```

Comment:
QQ if the annotations changed, does this mean Kueue now will only work with new AC?
Maybe we should support old annotations too?